db/schema.rb

```diff
@@ -11,16 +11,33 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150109005441) do
+ActiveRecord::Schema.define(version: 20150223135916) do
 
-  create_table "commands", force: true do |t|
-    t.text     "command",    limit: 16777215
+  create_table "commands", force: :cascade do |t|
+    t.text     "command",    limit: 10485760
```

Comment:
I'm guessing the recent upgrade of rails caused the activerecord schema generation to change the :force flag?
I don't know why the limit was the old value because it's very clear that it's supposed to be 10MB from the migration files.